android/src/main/kotlin/dev/rexios/polar/PolarPlugin.kt

```diff
@@ -568,33 +569,17 @@ class PolarWrapper(
         invoke("batteryLevelReceived", listOf(identifier, level))
     }
 
-    @Deprecated("", replaceWith = ReplaceWith(""))
-    override fun hrFeatureReady(identifier: String) {
-        // Do nothing
-    }
-
-    @Deprecated("", replaceWith = ReplaceWith(""))
-    override fun hrNotificationReceived(
+    override fun htsNotificationReceived(
         identifier: String,
-        data: PolarHrData.PolarHrSample,
+        data: PolarHealthThermometerData, 
     ) {
-        // Do nothing
-    }
-
-    @Deprecated("", replaceWith = ReplaceWith(""))
-    override fun polarFtpFeatureReady(identifier: String) {
-        // Do nothing
+       // Do nothing
```

Comment:
I was going to say this needs implemented, but I can't figure out what the iOS equivalent is so let's pretend we didn't see anything